src/utils/0x.test.ts

```diff
@@ -50,7 +50,7 @@ describe('ZeroExApi', () => {
     const chainId = 137
     const query = new URLSearchParams({
       buyAmount: ONE,
-      buyToken: DPI,
+      buyToken: '0x1494CA1F11D487c2bBe4543E90080AeBa4BA3C2b',
```

Comment:
just curious - why remove constant for the raw literal here?

Reply:
Once again being lazy. I first deleted to replace everything with USDC, until I noticed I don't need USDC in every test. Readded.